Commonalities/documentation/API-design-guidelines.md

```diff
@@ -1021,3 +1027,248 @@ Finally, this part describes the OAuth security applied to the API. This spec is
+    },
+    "uePort": 5060,
+    "eventType": "ROAMING_STATUS"
+  }
+}
+```
+
+response:
+
+```
+201 Created
+```
+```json 
+{
+  "notificationUrl": "https://application-server.com/v0",
+  "notificationAuthToken": "c8974e592c2fa383d4a3960714",
+  "subscriptionDetail": {
+    "ueId": {
+      "ipv4Addr": "192.168.0.1"
+    },
+    "uePort": 5060,
+    "eventType": "ROAMING_STATUS"
+  },
+  "subscriptionId": "456g899g",
+  "startsAt": "2023-03-17T16:02:41.314Z",
+  "expiresAt" : "2024-03-17T00:00:00.000Z"
+}
+```
+
+Note: If an API provides both pattern (indirect and resource-based), and an API customer requests both (instance base + subscription), the 2 requests should be handled  independently & autonomously. Depending on server implementation, it is acceptable, when the event occurred, that one or two notifications are sent to listener.
+
+
+### 12.2 Event notification
+
+The notification endpoint is used by the API server to notify the API consumer that an event occured.
+
+Note: The notification is the message posted on listener side. We describe the notification message in the CAMARA OAS but it could confusing because this endpoint should be implemented on the business API consumer side. This notice should be explicited mentioned in all CAMARA API documentation featuring notification.
+
+Only Operation POST is provided for `notifications` and the expected response code is `204`.
+
+For consistence among CAMARA API a uniform `notifications` model must be used:
+
+| name | type | attribute description | cardinality |
+| ----- |	-----  |	 -----  |  -----  | 
+| subscriptionId | string | subscription identifier - could be valued for Resource-based subscription | optional |
+| event | object | event structure - see next table | mandatory |
+
+Following table defines event attribute object structure: 
+
+| name | type | attribute description | cardinality |
+| ----- |	-----  |	 -----  |  -----  | 
+| eventId | string - uuid | Identifier of the event from the server where the event was reported | optional |
+| eventType | string | Type of event as defined in each CAMARA API. The event type are defined in UPPER_SNAKE_CASE| mandatory |
+| eventTime | string - datetime | Date time when the event occurred | mandatory |
+| eventDetail | object | A detailed event structure depending on the eventType | mandatory |
+
+Note: For operational and troubleshooting purposes it is relevant to accommodate use of X-Correlator header attribute. API listener implementations have to be ready to support and receive this data.
+
+Specific eventType "SUBSCRIPTION_ENDS" is defined to inform listener about subscrition termination. It is used when the subscription expire time (required by the requester) has been reached or if the API server has to stop sending notification prematurely. For this specific event, the `eventDetail` must feature `terminationReason` attribute.
+
+_Examples_
+
+Example for Roaming status - Request:
+
+```
+curl -X 'POST' \
```

Comment:
@eric-murray @PedroDiez @bigludo7 

Token neither used in the `header` or in the `data` for correlation

Reply:
Maybe the API consumer didn't provide `notificationAuthToken` for this particular subscription. It is optional, after all. However, I agree that including one in an `Authorization` header would be a better example.

Reply:
Agreed that the example will be better.
I've added it.